src/devices/Pn5180/samples/Program.cs

```diff
@@ -115,7 +115,7 @@ Pn5180 Ft4222()
 
     Ft4222Spi ftSpi = new Ft4222Spi(new SpiConnectionSettings(0, 1) { ClockFrequency = Pn5180.MaximumSpiClockFrequency, Mode = Pn5180.DefaultSpiMode, DataFlow = DataFlow.MsbFirst });
 
-    using GpioController gpioController = new(PinNumberingScheme.Board, new Ft4222Gpio());
+    GpioController gpioController = new(PinNumberingScheme.Board, new Ft4222Gpio());
```

Comment:
good catch!

Reply:
Well, it took me couple of seconds when I ran the sample to understand that there was a problem somewhere :-)